DenisZackharov/teacherss
# Update seeds architecture

### Summary

Now for create seeds for development need to execute `rails db:seed:developement`

### How it works

Screenshots/screencasts of the pull request introduced functionality.

### Test plan

List of steps to manually test introduced functionality:

1. Run `rails db:seed:developement`
2. Check that users with roles [super_admin, teacher, head_teacher, director] was created (if not existed already)

# Checklist:


### Deploy notes

Notes regarding deployment the contained body of work.
These should note any db migrations, ENV variables, services, scripts, etc.
